service/whatsapp/whatsapp.go

```diff
@@ -0,0 +1,167 @@
+package whatsapp
+
+import (
+	"encoding/gob"
+	"fmt"
+	"os"
+	"time"
+
+	qrcodeTerminal "github.com/Baozisoftware/qrcode-terminal-go"
+	"github.com/Rhymen/go-whatsapp"
+	"github.com/pkg/errors"
+)
+
+const (
+	qrLoginWaitTimeSeconds = 3
+	clientTimeout          = 5
+)
+
```

Comment:
```suggestion
)

var sessionFilePath = filepath.Join(os.TempDir(), "whatsappSession.gob")

```

Just to make sure we're always using the same path.